sdk/communication/azure-communication-administration/azure/communication/administration/_communication_identity_client.py

```diff
@@ -102,9 +102,8 @@ def create_user_with_token(
             List of scopes to be added to the token.
         :type scopes: list[str or 
             ~azure.communication.administration.models.CommunicationIdentityTokenScope]
-        :return: A CommunicationUser and a CommunicationIdentityToken tuple.
-        :rtype: tuple of (~azure.communication.administration.CommunicationUser,
-         ~azure.communication.administration.CommunicationIdentityToken)
+        :return: A CommunicationUser and a CommunicationIdentityAccessToken tuple.
+        :rtype: tuple of (~azure.communication.administration.CommunicationUser, ~azure.communication.administration.CommunicationIdentityAccessToken)
```

Comment:
as long as this is consistent in other parts of the code i'm fine with the formatting change

Reply:
Yeah, it does make it consistent with other parts of the code.